patches/blockprotocol+0.0.7.patch

```diff
@@ -67,3 +74,163 @@ index 20a9bbd..eb3a4e4 100644
  
  export type BlockProtocolUpdateLinksFunction = {
    (actions: BlockProtocolUpdateLinksAction[]): Promise<BlockProtocolLink[]>;
+@@ -452,4 +459,4 @@ export type BlockProtocolProps = {
+   linkedAggregations?: BlockProtocolLinkedAggregation[];
+   linkedEntities?: BlockProtocolEntity[];
+   linkGroups?: BlockProtocolLinkGroup[];
+-} & BlockProtocolFunctions;
++} & BlockProtocolFunctions & BlockProtocolStyling;
+diff --git a/node_modules/blockprotocol/styling.d.ts b/node_modules/blockprotocol/styling.d.ts
+new file mode 100644
+index 0000000..e7f656f
+--- /dev/null
++++ b/node_modules/blockprotocol/styling.d.ts
+@@ -0,0 +1,140 @@
++/**
++ * Block Protocol theme is inspired by MUI
++ * @see https://mui.com/customization/theming/
++ * @see 
```

Comment:
is there meant to be another link here?

Reply:
Yep, I missed it somehow. Thanks! https://github.com/hashintel/hash/pull/439/commits/b76590edc5629d31a776e09dc9a07229b1b22e0b